doc/testing-your-application.rst

```diff
@@ -67,15 +50,61 @@ You can override getters in your test class in the following way::
         }
     }
 
+VarnishTestCase
+------------------
+
+Configuration
+''''''''''''''''''
+
+By default, the ``VarnishTestCase`` starts and stops a Varnish server for you.
+All you have to do is to set your Varnish configuration (VCL) file. 
```

Comment:
you also need to set the hostname, no? or is the table incomplete and there is a default for the hostname?
/cc @ddeboer